src/main/java/pl/sda/weather/dao/ILocationDAO.java

```diff
@@ -1,9 +1,15 @@
 package pl.sda.weather.dao;
 
 
+import pl.sda.weather.model.LocationModel;
+
+import java.util.List;
 
 public interface ILocationDAO {
 
 
+    void addLocationModelToDB(LocationModel locationModel);
```

Comment:
maybe rename to "save model" -> interface shouldn't know terms like "file, database etc."